tex/gregoriotex.sty

```diff
@@ -125,12 +125,12 @@
 
 
 \def\grecoloredlines#1#2#3{%
-  \gre@warning{\protect\grecoloredlines\space is deprecated.\MessageBreak Use \protect\gresetlinecolor\space instead.}%
+  \gre@deprecated{\protect\grecoloredlines}{\protect\gresetlinecolor{red}}%
```

Comment:
I'm not sure if "red" should be mentioned here.  The command is not exactly equivalent in any case, so perhaps just leave the argument out, like the original warning?  I can't think of a good, terse way to do this here.